1.5/Dockerfile

```diff
@@ -0,0 +1,117 @@
+FROM openshift/base-centos7
+
+ARG ERLANG_VERSION=20.0
+ARG ELIXIR_VERSION=1.5.3
+ARG NODE_VERSION=8.11.2
+ARG NPM_VERSION=5.10.0
+ENV ERLANG_VERSION ${ERLANG_VERSION}
+ENV ELIXIR_VERSION ${ELIXIR_VERSION}
+ENV NODE_VERSION ${NODE_VERSION}
+ENV NPM_VERSION ${NPM_VERSION}
+
+LABEL io.k8s.description="Platform for building and running a phoenix app" \
+      io.k8s.display-name="build-phoenix" \
+      io.openshift.expose-services="4000:http" \
+      io.openshift.tags="builder,elixir,phoenix" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="/opt/app-root/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
```

Comment:
Same as `1.2`

Reply:
Removed in 551f804, I did accidentally copy that.